src/utils/icons.js

```diff
@@ -2,98 +2,98 @@ import styled from "styled-components";
 
 const StyledSvg = styled.svg``;
```

Comment:
This is not part of your user story, but why define a style component and leave it blank?

Reply:
I use it to experiment with svgs 😅